src/classes/ALLO_PaymentSync_SVC.cls

```diff
@@ -66,13 +72,28 @@ public class ALLO_PaymentSync_SVC {
         Decimal totalPaidUnpaidPayments = 0;
     }
 
+    private void addAllocationAmountToPayment(Id paymentId, Decimal allocAmount) {
```

Comment:
A minor thing. Can we add the Description part of the new methods just as other new methods?